Description: Do not use a server-based search engine for the doxygen documentation.
 As this would require a webserver with php enabled.
Author: Tobias Frost <tobi@debian.org>
Origin: <upstream|backport|vendor|other>, <URL, required except if Author is present>
Bug: <URL to the upstream bug report if any, implies patch has been forwarded, optional>
Bug-Debian: https://bugs.debian.org/914498
Forwarded: not-needed, Debian specific
Last-Update: 2020-04-08
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/dox/CMakeLists.txt
+++ b/dox/CMakeLists.txt
@@ -53,7 +53,7 @@
   get_filename_component (OCCT_ARTICLE_ABSOLUTE_PATH "${OCCT_OVERVIEW_DIR}/${OCCT_ARTICLE}" ABSOLUTE)
   get_filename_component (OCCT_ARTICLE_PARENT_DIR "${OCCT_ARTICLE_ABSOLUTE_PATH}" PATH)
 
-  if (EXISTS "${OCCT_ARTICLE_PARENT_DIR}/images")
+  if (EXISTS "${OCCT_ARTICLE_PARENT_DIR}/images" AND NOT "${OCCT_ARTICLE_PARAM_IMAGEPATH}" MATCHES ".*${OCCT_ARTICLE_PARENT_DIR}/images.*")
     set (OCCT_ARTICLE_PARAM_IMAGEPATH "${OCCT_ARTICLE_PARAM_IMAGEPATH} ${OCCT_ARTICLE_PARENT_DIR}/images")
   endif()
 
@@ -84,7 +84,7 @@
 
 # HTML Search engine options
 file (APPEND ${OCCT_CONFIG_FOR_DOXYGEN} "\nSEARCHENGINE           = YES")
-file (APPEND ${OCCT_CONFIG_FOR_DOXYGEN} "\nSERVER_BASED_SEARCH    = YES")
+file (APPEND ${OCCT_CONFIG_FOR_DOXYGEN} "\nSERVER_BASED_SEARCH    = NO")
 file (APPEND ${OCCT_CONFIG_FOR_DOXYGEN} "\nEXTERNAL_SEARCH        = NO")
 
 # Formula options
--- a/dox/resources/occt_ug_html.doxyfile
+++ b/dox/resources/occt_ug_html.doxyfile
@@ -4,7 +4,7 @@
 CREATE_SUBDIRS         = NO
 OUTPUT_LANGUAGE        = English
 ABBREVIATE_BRIEF       = 
-FULL_PATH_NAMES        = YES
+FULL_PATH_NAMES        = NO
 INHERIT_DOCS           = YES
 TAB_SIZE               = 4
 MARKDOWN_SUPPORT       = YES
